tabipy.py

```diff
@@ -220,6 +227,21 @@ def __init__(self, *rows):
             if index==0:
                 max_len = self.rows[0].column_count()
             
+    def cell(self, address, **kwargs):
+        """Allows for direct addressing of individual cells (row, column)
+
+        Any value not entered will remain unchanged.
+        Address is (row, column) with an origin index of 0."""
+        row, column = address
+        Row = self.rows[row]
+        cell = Row.cells[column]
+        if len(kwargs)>0:
```

Comment:
I'd still like to remove the kwargs from this, so it's just `t.cell(row, col)` returning a cell object.